### ISSUE TITLE
Do not pin Fandango dependencies

### ISSUE THREAD

**Issue by andreas-zeller** [fandango-fuzzer/fandango#115]
I suggest to _not_ pin dependencies in our `requirements.txt`, as this makes it harder for fandango to co-exist with other packages. I'd rather go and assume the newest version(s), and only request/exclude specific versions if we know there is a problem with some version.

Of course, I know about virtual python envs and what not, but installing these will be a pain for novice users; plus, there's many of them, and I would rather keep the tutorial simple (notably, the installation instructions). Plus, users may report issues due to pinned versions to us, and we don't want resolving these.

TL;DR: Having versions pinned may avoid some problems, but will introduce more problems for our users (and for us).

Comments welcome!
